src/zenml/utils/dashboard_utils.py

```diff
@@ -34,6 +35,13 @@ def get_base_url() -> Optional[str]:
     client = Client()
 
     if client.zen_store.type == StoreType.REST:
+        # if the server config has a base URL use that
+        server_config = ServerConfiguration.get_server_config()
+        if server_config.base_url:
+            return (
+                server_config.base_url
```

Comment:
If the base URL is a cloud one, there is no `/workspaces/...` right? Maybe we need to differentiate here whether this is a cloud tenant (I guess if the base url contains `cloud.zenml.io`?) until we open-source the cloud dashboard and can unify these?

Reply:
good catch, i've added a check